tests/unit/aiplatform/test_datasets.py

```diff
@@ -106,6 +106,21 @@
 # misc
 _TEST_OUTPUT_DIR = "gs://my-output-bucket"
 
+_TEST_TABULAR_DATASET_LIST = [
```

Comment:
To test the filtering, I think we should add at least one item in the mock that isn't tabular.

Reply:
Great call out, added a text and image dataset into the mix.